src/main/java/seedu/address/ui/ModuleCard.java

```diff
@@ -56,8 +56,8 @@ public ModuleCard(Module module, int displayedIndex) {
         moduleCode.setText(module.getModuleCode().moduleCode);
         lectureDetails.setText(module.getLectureDetails().value);
         tutorialDetails.setText(module.getTutorialDetails().value);
-        lectureZoomLink.setText(module.getZoomLink().zoomLink);
-        tutorialZoomLink.setText(module.getZoomLink().zoomLink);
+        lectureZoomLink.setText(module.getLectureZoomLink().zoomLink.orElse(""));
```

Comment:
should we allow the UI class to decide whether we use an empty string? I think it should be handled by the getLectureZoomLink method